### PR TITLE
Update occurrence: result indicator @value attributes

### PR DESCRIPTION
**Pull request by allthatilk**
Fixes the schema aspects of #353

### CODE REVIEW

File: tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_value/attribute-exists.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    </activity-date>
+    <result type="xx" aggregation-status="0">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <indicator measure="xx" ascending="0">
+        <title>
+          <narrative>Sample text</narrative>
+        </title>
+        <baseline year="2017" value= "xx"><!-- PASS: optional attribute included once -->
```
Comment (dalepotter) [inline tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_value/attribute-exists.xml]: Remove whitespace for consistency: `value= "xx"` -> `value="xx"`

**Review by dalepotter — CHANGES_REQUESTED** [review]
Small thing on whitespace...

File: tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_iso-date/attribute-does-not-exist.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    </activity-date>
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <indicator measure="xx">
+        <title>
+          <narrative>Sample text</narrative>
+        </title>
+        <baseline year="2017" value= "xx"><!-- PASS: optional iso-date attribute not included -->
```
Comment (dalepotter) [inline tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_iso-date/attribute-does-not-exist.xml]: Also here: `value= "xx"` -> `value="xx"`

File: tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_iso-date/attribute-exists-for-element-correct-date-format.xml [hunk 3]
```diff
@@ -24,7 +24,7 @@
         <title>
           <narrative>Sample text</narrative>
         </title>
-        <baseline year="2017" value= "xx" iso-date="2017-11-16"><!-- PASS: iso-date attribute included using correct datetime format -->
+        <baseline year="2017" value= "xx" iso-date="2017-11-16"><!-- PASS: optional iso-date attribute included using correct datetime format -->
```
Comment (dalepotter) [inline tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/_iso-date/attribute-exists-for-element-correct-date-format.xml]: Also here: `value= "xx"` -> `value="xx"`

**Comment by allthatilk** [comment]
@dalepotter ready for rereview =)

**Review by dalepotter — APPROVED** [review]
Looks good!

**Pull request merged**